### PR TITLE
add ac-3,ec-3 and flac codecs gst caps

### PR DESCRIPTION
**Pull request by emutavchi**
Allow discovery and playback of mentioned audio codecs with MSE / EME player.

### CODE REVIEW

**Comment by woutermeek** [comment]
@emutavchi we will add this one after we've upstreamed and added  https://github.com/WebPlatformForEmbedded/WPEWebKit/pull/797

**Review by eocanha — COMMENTED** [review]
The changes look fine. I've upstreamed them here:
- Bug [237843](https://bugs.webkit.org/show_bug.cgi?id=237843)
I'll merge them back after upstream landing.

**Review by eocanha — APPROVED** [review]

**Comment by eocanha** [comment]
I had to revert the commit and push it again on wpe-2.28 because I forgot to add a local change to the merged commit. Sorry.

**Pull request merged**
